swift_kernel.py

```diff
@@ -423,7 +423,7 @@ def _process_installs(self, code):
         user_install_location = None
         for index, line in enumerate(code.split('\n')):
             line = self._process_system_command_line(line)
-            line, install_location = self._process_install_location_line(line)
+            line, install_location = self._process_install_location_line(line_index, line)
```

Comment:
Should `line_index` actually be `index` here?

Reply:
```suggestion
            line, install_location = self._process_install_location_line(index, line)
```